Hi @pinge thanks for the pull request!  I don't regularly use yarn and wanted to try it out with the wdio-eslinter-service, so I replaced it in my node_modules folder with your pull request.  Here's what I see when running wdio. I don't see any of the eslint output:

```
Execution of 89 workers started at 2022-03-02T11:44:32.168Z

2022-03-02T11:44:32.216Z WARN wdio-eslinter-service: initialize wdio-eslinter-service using yarn runner.
wdio.conf.js:318: <log>: Deleted files and directories:/n /Users/james/Dev/project/allure-results
yarn run v1.22.17
$ eslint . --quiet
✨  Done in 4.10s.
[0-0] For help, see: https://nodejs.org/en/docs/inspector
[0-0] 2022-03-02T11:44:37.611Z WARN @wdio/utils:shim: You are running tests with @wdio/sync which will be discontinued starting Node.js v16.Read more on https://github.com/webdriverio/webdriverio/discussions/6702
[0-0] RUNNING in chrome - test/spec/test.spec.js
```

### Here's my setup:

 **wdio.conf.js:**
```
services: ['chromedriver', [
        'eslinter',
        {
            runnerType: 'yarn',
            scriptName: 'yarneslint',
        }
    ]],
```

**package.json**:
```
...
"scripts": {
    "yarneslint": "eslint ."
},
...
```

If I run the script with just yarn, I see tons of eslint warnings as expected:

```
$ yarn yarneslint
yarn run v1.22.17
$ eslint .

/Users/james/Dev/projects/test/specs/test.spec.js:
   20:5  warning  Unexpected console statement  no-console
   59:5  warning  Unexpected console statement  no-console
   95:5  warning  Unexpected console statement  no-console
  122:5  warning  Unexpected console statement  no-console
...
```

When I removed `--quiet` from [this line in eslint-yarn-runner.js](https://github.com/box-to-box/wdio-eslinter-service/blob/c37bd4b73897c022d4f1736a6a7f7f077f3fda5f/src/eslint-yarn-runner.js#L8) I then got the output.  Do you think that `--quiet` should be removed or should it stay?

Thank you!
James

--- a/src/eslint-yarn-runner.js
+++ b/src/eslint-yarn-runner.js
@@ -5,7 +5,7 @@ const { spawn } = require('child_process');
 module.exports = function (scriptName) {
     return new Promise((resolve, reject) => {
         const yarnCmd = process.platform.match('^win') !== null ? 'yarn.cmd' : 'yarn';
-        const eslint = spawn(yarnCmd, [scriptName, '--quiet'], { stdio: "inherit" });
+        const eslint = spawn(yarnCmd, ['run', '--silent', scriptName], { stdio: "inherit" });
 
         eslint.on('close', (code) => {
             if (code !== 0)

